test/test_tb2bd.cc

```diff
@@ -9,6 +9,7 @@
 #include "print_matrix.hh"
 #include "grid_utils.hh"
 #include "scalapack_support_routines.hh"
+#include "internal/internal.hh"
```

Comment:
Do we need `internal.hh`? It's best for testers not to rely on internals.

Reply:
Right, it is not needed. I will delete it.